File: web/filing/models/income/salary/employer-address-model.ts
export interface EmployerAddressModel {
    employerAddressId?: number | null
    employerId?: number | null
    addressLine1: string
    addressLine2?: string | null
    landmark?: string | null
    city: string
    district?: string | null
    state: string
    pincode?: string | null
    country: string
  }

File: api/filing/models/employer_address_model.py
from .api_base_model import ApiBaseModel


class EmployerAddressModel(ApiBaseModel):
    employer_address_id: int | None = None
    employer_id: int | None = None
    address_line1: str
    address_line2: str | None = None
    landmark: str | None = None
    city: str
    district: str | None = None
    state: str
    pincode: str
    country: str
# EmployerAddressModel
